create.js

```diff
@@ -184,6 +150,7 @@ module.exports = function create (path, opts, keys) {
     })
   }
 
+  /* via last index */
   db.getVectorClock = function (_, cb) {
```

Comment:
in several files I've collected methods which depend on a particular view into one place, to make them easier to track + maintain